pkg/apis/acme/v1alpha2/types_issuer.go

```diff
@@ -64,10 +67,12 @@ type ACMEExternalAccountBinding struct {
 	// the External Account Binding keyID above.
 	// The secret key stored in the Secret **must** be un-padded, base64 URL
 	// encoded data.
+	// +kubebuilder:validation:Required
```

Comment:
This change doesn't seem to do anything - fields are already considered `required` if they are not `optional`